src/main/java/com/microsoft/azure/springcloudplayground/dependency/Dependency.java

```diff
@@ -90,11 +121,10 @@
 
     private List<String> keywords = new ArrayList<>();
 
+    @Getter
+    @Setter
     private List<Link> links = new ArrayList<>();
 
-    public Dependency() {
```

Comment:
Is this constructor needed?

Reply:
Needed, `@NoArgsConstructor` will cover this.